ledger-test-tool/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_15/ExplicitDisclosureIT.scala

```diff
@@ -527,11 +530,17 @@ object ExplicitDisclosureIT {
       val request = delegateParticipant
         .submitAndWaitRequest(
           delegate,
-          Command.of(
-            Command.Command.Create(
-              CreateCommand(
-                Some(Dummy.id.unwrap),
-                Some(Dummy(delegate).arguments),
+          JList.of(
+            CommandJava.fromProtoCommand(
```

Comment:
Why can't we stay in the Java world throughout this construction?

Reply:
a6c1579f5761a2684995c08ff29be1ef46cc9cf6